<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectorWidget</class>
 <widget class="QWidget" name="SelectorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>209</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>selectorWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QPushButton" name="selectorButton">
     <property name="text">
      <string>Select</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="selectionLabel">
     <property name="text">
      <string>No Item Selected</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ExpandingLabelWidget" name="extraDescriptionLabel" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ExpandingLabelWidget</class>
   <extends>QWidget</extends>
   <header>..util.qtUtil</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
